File: backend/src/main/java/com/mohamed/tahiri/backend/conversation/ConversationTitle.java
package com.mohamed.tahiri.backend.conversation;

public class ConversationTitle {
    public Long id;
    public String fullName;
    public String lastMessage;
    public String time;
    public String image;
    public Long admin;
    public Boolean isAllRead;
    public Long lastSender;

    public ConversationTitle(Long id, String fullName, String lastMessage, String time, String image, Long admin, Boolean isAllRead, Long lastSender) {
        this.id = id;
        this.fullName = fullName;
        this.lastMessage = lastMessage;
        this.time = time;
        this.image = image;
        this.admin = admin;
        this.isAllRead = isAllRead;
        this.lastSender = lastSender;
    }

    @Override
    public String toString() {
        return super.toString();
    }
}

File: android/app/src/main/java/com/mohamed/tahiri/android/model/ConversationTitle.kt
package com.mohamed.tahiri.android.model

data class ConversationTitle(
    val id: Long,
    val fullName: String,
    val lastMessage: String,
    val time: String,
    val image: String,
    val admin: Long,
    val isAllRead: Boolean,
    val lastSender:Long
)